src/components/ChatEntry.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
-      </section>
-    </div>
-  );
+    const likeButton = () => {
+        const updatedChatEntry = {
+            id: props.id,
+            sender: props.sender,
+            body: props.body,
+            timeStamp: props.timeStamp,
+            liked: !props.liked,
+        };
+        props.onUpdateChatEntry(updatedChatEntry);
```

Comment:
Nice use of your passed-in callback to allow parent-owned logic to run when the like button is clicked.